packages/kbn-repo-packages/modern/plugins.js

```diff
@@ -107,11 +107,7 @@ function getPluginPackagesFilter(selector = {}) {
  */
 function getDistributablePacakgesFilter() {
   return (pkg) => {
-    if (
-      pkg.isDevOnly ||
-      pkg.manifest.type === 'functional-tests' ||
-      pkg.manifest.type === 'test-helper'
-    ) {
+    if (pkg.isDevOnly()) {
```

Comment:
Any context on this?  It looks like the distribution is ~11mb larger.

[Before](https://buildkite.com/elastic/kibana-on-merge/builds/33399#0189b0ed-f22a-4ad6-90b3-f6cc941cab5f)
[After](https://buildkite.com/elastic/kibana-pull-request/builds/146169#0189ad76-6e34-43b3-8699-56a7d87bfc50)

Reply:
:+1: thanks for checking